moira/api/resources/targets.py

```diff
@@ -0,0 +1,24 @@
+from twisted.internet import defer
+
+from moira.api.request import delayed
+from moira.api.resources.redis import RedisResouce
+
+
+class Targets(RedisResouce):
+
+    def __init__(self, db):
+        RedisResouce.__init__(self, db)
+
+    def getChild(self, path, request):
+        if not path:
+            return self
+
+    @delayed
+    @defer.inlineCallbacks
+    def render_GET(self, request):
+        targets = yield self.db.getTargets()
+        if 'search' in request.args.keys():
```

Comment:
This is implicit behavior.
Better return 400 (bad request) code to client if required argument 'search' is missing.
Also 'search' may be not the best name for argument.